tinymce/tests/test_widgets.py

```diff
@@ -58,6 +58,14 @@ def test_config_from_language_code(self):
                 config = get_language_config()
                 self.assertEqual(config["language"], lang_expected)
 
+    def test_language_override_from_config(self):
+        langs = ["es_ES", "ru"]
+        for lang in langs:
+            tinymce.settings.DEFAULT_CONFIG.update(language=lang)
+            config = get_language_config()
+            self.assertEqual(config["language"], lang)
+        tinymce.settings.DEFAULT_CONFIG.pop("language")
```

Comment:
I did try to mock `DEFAULT_CONFIG` instead of changing it, but I was not able to make it work.
The only way I can think about of making it possible would be to pass it as an argument to `get_language_config()` like `content_language` was done. What are your thoughts?

Reply:
Sure, thank you for your feedback @claudep!
By the way, the built-in `to_locale()` was a great improvement, had no idea about it 👍